src/Extensions/Diagnostics.HealthChecks/Internal/RestHealthCheckPublisher.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Omex.Extensions.Abstractions.ExecutionContext;
+using Microsoft.ServiceFabric.Client;
+using Microsoft.ServiceFabric.Client.Http;
+using sfc = Microsoft.ServiceFabric.Common;
+using sfh = System.Fabric.Health;
+
+namespace Microsoft.Omex.Extensions.Diagnostics.HealthChecks
+{
+	/// <summary>
+	/// Just mock
+	/// </summary>
+	public class RestHealthCheckPublisher : HealthCheckPublisher
+	{
+		private readonly ServiceFabricHttpClient m_client;
+		private readonly string m_serviceId;
+		/// <summary>
+		/// 
+		/// </summary>
+		protected override string HealthReportSourceIdImpl => nameof(RestHealthCheckPublisher);
+
+		/// <summary>
+		/// 
+		/// </summary>
+		public RestHealthCheckPublisher(Uri clusterEndpoints, string serviceId) : base()
+		{
+			m_serviceId = serviceId;
+			m_client = (ServiceFabricHttpClient)new ServiceFabricClientBuilder()
+				.UseEndpoints(clusterEndpoints)
+				.BuildAsync().GetAwaiter().GetResult();
+		}
+
+
+		/// <summary>
+		/// 
+		/// </summary>
+		/// <param name="healthInfo"></param>
+		/// <returns></returns>
+		private sfc.HealthInformation fromSfHealthInformation(sfh.HealthInformation healthInfo)
```

Comment:
The naming convention for .NET requires starting method name from capital letter. https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers